### ISSUE TITLE
[Req] Torrentgalaxy

### ISSUE THREAD

**Issue by whitesnakeftw** [Jackett/Jackett#3792]
Url: https://torrentgalaxy.org/

Very good open tracker for newest content.

**Comment by kaso17** [comment]
If you know a little HTML/CSS you can try to add it yourself.
See https://github.com/Jackett/Jackett/wiki/Definition-format for details.
If you've any questions feel free to ask.

If the tracker already provides some kind of API (e.g. gazelle based trackers) please let us know. Then the Cardigann definition approach might be not not the best solution

**Comment by garfield69** [comment]
https://torrentgalaxy.to/torrents.php?search= returns no results
https://torrentgalaxy.to/latest has a different layout
may need to resort to .Today.Year ?

**Comment by garfield69** [comment]
#15358
- https://torrentgalaxy.to/ TGX is offline. Updates will be posted in case of any changes.
- now the caption reads: Error 4ever?

**Comment by vdbhb59** [comment]
Yes, the official torrentgalaxy.to; tgx.sb; proxygalaxy,me (status+mirror) are all down. Do not remove it yet, as the domains are not seized, and this seems to be some other issue. As I said, I am active on reddit and continuously checking the domains for changes.

**Comment by vdbhb59** [comment]
It is up now, with new URL: https://torrentgalaxy.mx/ This is where every official URL is automatically redirecting to. I will raise a PR for the update to URL as well.

**Comment by garfield69** [comment]
online. but not accessible to Jackett until #15358 is resolved.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://torrentgalaxy.to/ , https://torrentgalaxy.mx/ , https://tgx.rs/ > ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by vdbhb59** [comment]
> online

Yeah it was down for more than 15 hours yesterday.

**Comment by garfield69** [comment]
https://tgx.rs/ and https://torrentgalaxy.to/ > ERR_CONNECTION_REFUSED
All uptime checkpoints fail with TCP Connection Error

**Comment by vdbhb59** [comment]
Yeah, all sites are down. The status page though states otherwise.
I even tried the onion link, same.

**Comment by garfield69** [comment]
https://torrentgalaxy.to/ online
https://tgx.rs/ > connection refused
https://torrentgalaxy.mx/ online

**Comment by vdbhb59** [comment]
Yeah, it was down and when it came back the database was not working.
Reddit was flaming lols. This has been every 3-4 weeks since few months now. I am a regular there, but unfortunately only the owner has the actual info, which he or she never shares.

**Comment by garfield69** [comment]
https://torrentgalaxy.to/ , https://torrentgalaxy.mx/ > cloudflare Bad gateway Error code 502

**Comment by garfield69** [comment]
v0.22.1726
